api/printful/printful.php

```diff
@@ -1,19 +1,19 @@
 <?php
-
+require('../api-cache.php');
 class PrintfulCustomAPI {
     private $api_key;
-
+    private const PRINTFUL_CACHE_CONFIG_ID = 1;

```

Comment:
Config ID should be unnecessary in the code, we can look everything up using the endpoint URL.

Reply:
Removed in c289fad. Using the retrieved config ID on [line 82](https://github.com/untrobotics/website/pull/158/commits/c289fad3a479b93eff024e7c5094ab293f1e3a7b#diff-970b238464763356bddea5f27fef2cba1f6162b327686a4fe165921e8bb448e9R82)